File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i = 0;
        int j = numbers.length - 1;
        while(i < j){
            int x = numbers[i] + numbers[j];
            if(x == target) break;
            else if(x < target) i++;
            else j--;
        }
        i++; j++;
        return new int[]{i , j};
        
    }
}

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        numMap = {}
        for i, num in enumerate(numbers):
            tar = target - num
            if tar in numMap:
                return [numMap[tar] + 1, i + 1]
            numMap[num] = i
            